src/components/SaveJokes/SaveJokes.tsx

```diff
@@ -0,0 +1,54 @@
+import { useTranslation } from "react-i18next";
+import classes from "./SaveJokes.module.scss";
+import JokesAmountPicker from "../JokesAmountPicker/JokesAmountPicker";
+import { useState } from "react";
+import { useDownloadJokes } from "../../hooks/useDownloadJokes";
+import AlertSnackbar from "../AlertSnackbar/AlertSnackbar";
+
+const SaveJokes = () => {
+  const [isValid, setIsValid] = useState(true);
```

Comment:
should be described what is valid - if it's jokes then it should be  `isJokesAmountValid`